src/lib/smartdc/mdata-image

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2015, Joyent, Inc. All rights reserved.
+#
+# Execute metadata image scripts
+
+# load common functions and vars
+. ./lib/smartdc/common.lib
```

Comment:
Can you change this so it sources the absolute path?

e.g. `. /lib/smartdc/common.lib`